### PR TITLE
Add ODH ClusterServiceVersion for 0.8.0 release

### LINKED ISSUE #54: ODH 0.8 Release

### PR DESCRIPTION
**Pull request by LaVLaS**
ODH release for 0.8 release. 
The only major update that I added to the description was the support for JupyterLab notebooks in JupyterHub

Closes #54

### CODE REVIEW

File: deploy/olm-catalog/opendatahub/0.8.0/opendatahub-operator.v0.8.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,364 @@
...
+      version: v1
+      displayName: Open Data Hub
+      group: kfdef.apps.kubeflow.org
+  description: |
+    The Open Data Hub is a machine-learning-as-a-service platform built on Red Hat's Kubernetes-based OpenShift® Container Platform, Ceph Object Storage, and integrating a collection of open source projects.
+
+    Open Data Hub is a meta-project that integrates open source projects into a practical solution. It aims to foster collaboration between communities, vendors, user-enterprises, and academics following open source best practices. The open source community can experiment and develop intelligent applications without incurring high costs and having to master the complexity of modern machine learning and artificial intelligence software stacks.
+
+    ### Components
+    * JupyterHub v3.0.7 - Open source multi-user notebook platform w/ GPU support. ***NEW*** Support for JupyterLab notebooks
```
Comment (LaVLaS) [inline deploy/olm-catalog/opendatahub/0.8.0/opendatahub-operator.v0.8.0.clusterserviceversion.yaml:253]: @vpavlin Is there any documentation for the new `JupyterLab` notebook feature?

**Comment by nakfour** [comment]
@LaVLaS is it possible to add to the description something about the new example kfctl.yaml files that show the mixing of ODH with KF. We have one for mixing in pytorch, tfjob to ODH and one for adding prom + grafana to Kubeflow.

File: deploy/olm-catalog/opendatahub/0.8.0/opendatahub-operator.v0.8.0.clusterserviceversion.yaml [hunk 2]
```diff
@@ -0,0 +1,364 @@
...
+    - description: KfDef is the Schema for the applications API
+      kind: KfDef
+      name: kfdefs.kfdef.apps.kubeflow.org
+      version: v1
+      displayName: Open Data Hub
+      group: kfdef.apps.kubeflow.org
+  description: |
+    The Open Data Hub is a machine-learning-as-a-service platform built on Red Hat's Kubernetes-based OpenShift® Container Platform, Ceph Object Storage, and integrating a collection of open source projects.
+
+    Open Data Hub is a meta-project that integrates open source projects into a practical solution. It aims to foster collaboration between communities, vendors, user-enterprises, and academics following open source best practices. The open source community can experiment and develop intelligent applications without incurring high costs and having to master the complexity of modern machine learning and artificial intelligence software stacks.
```
Comment (vpavlin) [inline deploy/olm-catalog/opendatahub/0.8.0/opendatahub-operator.v0.8.0.clusterserviceversion.yaml:250]: We need to fix the `meta-project` term wherever it appears - it does not make sense to me:) ODH is a project, but it is a meta-operator

File: deploy/olm-catalog/opendatahub/0.8.0/opendatahub-operator.v0.8.0.clusterserviceversion.yaml (hunk 1 above)
Comment (vpavlin) [reply deploy/olm-catalog/opendatahub/0.8.0/opendatahub-operator.v0.8.0.clusterserviceversion.yaml:253]: Hmm..not really..if you want to highlight something, maybe highlight the addition of Elyra (with JupyterLab) ? Something like

```
***NEW*** New notebook for Elyra (using JupyterLab interface)
```

File: deploy/olm-catalog/opendatahub/0.8.0/opendatahub-operator.v0.8.0.clusterserviceversion.yaml [hunk 3]
```diff
@@ -0,0 +1,364 @@
...
+
```
Comment (vpavlin) [inline deploy/olm-catalog/opendatahub/0.8.0/opendatahub-operator.v0.8.0.clusterserviceversion.yaml:251]:
```suggestion

    Open Data Hub operator allows users to install and manage Open Data Hub and Kubeflow components. Users can mix and match tools from each project to fulfill the needs of their use case. Each of the projects share some components, but canbe mostly seens as an extension of each other to provide as complete solution as possible for both hobby and enterprise users.
```

Comment (vpavlin) [reply deploy/olm-catalog/opendatahub/0.8.0/opendatahub-operator.v0.8.0.clusterserviceversion.yaml:251]: How about adding something like this ^ @crobby @nakfour @LaVLaS ?

File: deploy/olm-catalog/opendatahub/0.8.0/opendatahub-operator.v0.8.0.clusterserviceversion.yaml (hunk 1 above)
Comment (vpavlin) [reply deploy/olm-catalog/opendatahub/0.8.0/opendatahub-operator.v0.8.0.clusterserviceversion.yaml:253]:
```suggestion
    * JupyterHub v3.0.7 - Open source multi-user notebook platform w/ GPU support. ***NEW*** Notebook with support for Elyra and JupyterLab
```

File: deploy/olm-catalog/opendatahub/0.8.0/opendatahub-operator.v0.8.0.clusterserviceversion.yaml (hunk 3 above)
Comment (LaVLaS) [reply deploy/olm-catalog/opendatahub/0.8.0/opendatahub-operator.v0.8.0.clusterserviceversion.yaml:251]: Typo fix:

`...but can be mostly seen as an extension of each other to provide as complete a solution as possible for both hobby and enterprise users.`

Comment (vpavlin) [reply deploy/olm-catalog/opendatahub/0.8.0/opendatahub-operator.v0.8.0.clusterserviceversion.yaml:251]: Updated the suggestion

File: deploy/olm-catalog/opendatahub/0.8.0/opendatahub-operator.v0.8.0.clusterserviceversion.yaml (hunk 1 above)
Comment (LaVLaS) [reply deploy/olm-catalog/opendatahub/0.8.0/opendatahub-operator.v0.8.0.clusterserviceversion.yaml]: Updated

File: deploy/olm-catalog/opendatahub/0.8.0/opendatahub-operator.v0.8.0.clusterserviceversion.yaml (hunk 3 above)
Comment (LaVLaS) [reply deploy/olm-catalog/opendatahub/0.8.0/opendatahub-operator.v0.8.0.clusterserviceversion.yaml:252]: Updated

**Comment by LaVLaS** [comment]
> @LaVLaS is it possible to add to the description something about the new example kfctl.yaml files that show the mixing of ODH with KF. We have one for mixing in pytorch, tfjob to ODH and one for adding prom + grafana to Kubeflow. My suggestions would be after this sentence "To deploy Kubeflow, use this example." Add something like, "To deploy Kubeflow with monitoring, use this kfdef example that adds Prom and Grafana. To add distributed learning tools to ODH such as pytorchjobs and tf-jobs use this example."

@nakfour Take a look, I added a section to the description titled `Mixing ODH and Kubeflow` with a short summary and a link to the opendatahub.io docs on [Combining Open Data Hub and Kubeflow Components](http://opendatahub.io/docs/kubeflow/mixing.html)

**Comment by nakfour** [comment]
that looks good @LaVLaS thank you

**Review by nakfour — APPROVED** [review]

File: deploy/olm-catalog/opendatahub/0.8.0/opendatahub-operator.v0.8.0.clusterserviceversion.yaml (hunk 2 above)
Comment (LaVLaS) [reply deploy/olm-catalog/opendatahub/0.8.0/opendatahub-operator.v0.8.0.clusterserviceversion.yaml]: Fixed via the comment below to update the description

**Review by crobby — APPROVED** [review]

**Pull request merged**